docs/geedocs/answer/4492599.html

```diff
@@ -90,7 +93,7 @@
 <p><strong>Browsers</strong></p>
 <p>Google Earth Enterprise is supported on the following browsers on all operating systems:</p>
 <ul>
-<li dir="ltr">Google Chrome version 10 and later (32-bit)</li>
+<li dir="ltr">Google Chrome version 10 and later</li>
```

Comment:
There is a list of supported OSs in the main README.md on github, but I don't know of a list of supported browsers.  For that matter, we aren't doing any browser testing anyway, so we don't know for sure which ones work.  You're right that it will be hard to keep this documentation updated, but I think it's OK not to solve that right now.